src/scenes/PlantingSitesRouter/view/GenericZoneView.tsx

```diff
@@ -153,5 +158,9 @@ const DetailsRenderer =
       return <CellRenderer {...props} value={numMonitoringPlots > 0 ? numMonitoringPlots : ''} sx={textStyles} />;
     }
 
+    if (column.key === 'areaHa') {
+      return <CellRenderer {...props} value={value ? value : ''} sx={textStyles} />;
+    }
```

Comment:
Nit - this can be simplified from `value={value ? value : ''}` to `value={value || ''}`, unless value can be 0 or false in which case you could change it to `value={value ?? ''}` (which means use `''` only if `value` is null or undefined but keeps 0 and false

Reply:
Good catch